templates/_call-stack-item.php

```diff
@@ -2,40 +2,28 @@
 /* @var $file string|null */
 /* @var $line int|null */
 /* @var $class string|null */
-/* @var $method string|null */
+/* @var $function string|null */
 /* @var $index int */
 /* @var $lines string[] */
 /* @var $begin int */
 /* @var $end int */
 /* @var $args array */
-/* @var $this \Yiisoft\ErrorHandler\HtmlRenderer */
-
-$html = <<<HTML
-IDE
-<svg class="icon icon--new-window" focusable="false" aria-hidden="true" width="16" height="16">
-    <use href="#new-window"></use>
-</svg>
-HTML;
+/* @var $this \Yiisoft\ErrorHandler\Renderer\HtmlRenderer */
 ?>
-<li class="<?= ($index === 1 || !$this->isCoreFile($file)) ? 'application' : '' ?> call-stack-item"
-    data-line="<?= (int) ($line - $begin) ?>">
+<li class="<?=!empty($lines) ? 'application' : '' ?> call-stack-item" data-line="<?= (int) ($line - $begin) ?>">
```

Comment:
I think we can still check for `isCoreFile()` somehow such as by checking for vendor/yiisoft in file path.

Reply:
I don't understand why this is necessary, this check decides whether to disclose the call stack content or not. Therefore, checking the existence of the content is the most logical.

Reply:
The idea is that framework is well-covered with tests so displaying trace from parts of the framework will less likely help to solve the problem. The problem is usually in the application itself so we make default state to focus on these.

Reply:
OK